src/app/static/src/app/components/downloadResults/UploadModal.vue

```diff
@@ -112,12 +112,8 @@
                 return formatDateTime(date)
             },
             setDefaultCheckedItems: function () {
-                const uploadFilesKeys = ["outputZip", "outputSummary"]
-                uploadFilesKeys.forEach(key => {
-                    if (this.uploadFiles.hasOwnProperty(key)) {
-                        this.uploadFilesToAdr.push(key)
-                    }
-                })
+                this.uploadFilesToAdr = ["outputZip", "outputSummary"]
+                    .filter(key => this.uploadFiles.hasOwnProperty(key))
```

Comment:
Looks like we're lacking a test that relies on this, but should be easy to remedy?

Reply:
There is a test called "checkboxes are set by default" but I don't know how to test a change in uploadFiles as it comes from the adr state.

Reply:
I believe I've found a solution now.